src/frontend/main.go

```diff
@@ -187,6 +225,23 @@ func initTracing(log logrus.FieldLogger) {
 	// trace.ProbabilitySampler set at the desired probability.
 	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
 	initStackdriverTracing(log)
+
+	// Initialize OpenTelemetry Tracer
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	exporter, err := texporter.NewExporter(texporter.WithProjectID(projectID))
+	if err != nil {
+		log.Fatalf("failed to initialize exporter: %v", err)
+	}
+
+	// Create trace provider with the exporter.
+	tp, err := sdktrace.NewProvider(sdktrace.WithConfig(
+		sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithSyncer(exporter),
+		sdktrace.WithResource(resource.New(standard.ServiceNameKey.String("FrontendService"))))
```

Comment:
Isn't the package picking up the resource automatically? It should but it's possible that it's not implemented yet.
If it's not implemented yet, worth changing this to be "GKE". The resource name is supposed to be a platform on which the code is running.

Reply:
Not 100% sure, but I changed it to GKE